directory/DirectoryBuilder.js

```diff
@@ -93,6 +93,7 @@ class DirectoryBuilder {
       code: details.code,
       issuer: details.issuer,
       domain: anchorDomain,
+      instructions: details.instructions || "Coming Soon...",
```

Comment:
This is too opinionated and is promising something is coming soon.

Instructions should only be in an asset if it is set. If not set, then instructions should not appear in the asset object.